_develop/webpack.config.js

```diff
@@ -85,6 +85,7 @@ const tsRules = {
 
 const baseConfig = {
   mode: 'development',
```

Comment:
We could set this to production - or do we have some advantage from development?